www-apps/redmine/redmine-3.2.8.ebuild

```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
You can bump to EAPI=6 so that all ebuilds have the same EAPI this makes maintenance easier